docs/help/service-security.md

```diff
@@ -0,0 +1,147 @@
+# Service Security
+
+Secured [service](./definitions.md#microservice-or-service) providing is a key in a loosley-coupled, System-Of-Systems environment. Arrowhead 5 offers multiple security levels to be applied by the service providers.
+
+**Authorization Rules:**<br />
+Specifies which systems have what permissions to which service instances.
+> Learn more about the [authorization rules](../core_systems/authorization.md).
+
+**Security Policies:**<br />
+Specifies how to prove access rights when consuming a service.
+> Learn more about the [security policies](#security-policies).
+
+**Communication Profiles:**<br />
+Among other things, specifies the required encryption for data exchange and the method of providing service access credentials, like certificates or tokens.
+> Learn more about the [communication profiles](../api/communication-profiles/communication-profiles-overview.md)
+
+## Security Policies
+
+The Service Security Policy must be defined during service registration for each [interface](./definitions.md#service-interface) of the given [service](./definitions.md#microservice-or-service) by every provider System.
```

Comment:
by every Provider System -> that is not true. Service registration is possible by 3rd party as well via management endpoints. So I'd just remove this part of the sentence.